test/evp_test.c

```diff
@@ -20,8 +20,9 @@
 #include "internal/numbers.h"
 #include "testutil.h"
 
-/* Remove spaces from beginning and end of a string */
-
+/*
+ * Remove spaces from beginning and end of a string
+ */
```

Comment:
That's new...  not sure I see why you'd make a block comment from a one liner, but perhaps that's becoming the general rule outside of functions?